package.json

```diff
@@ -24,9 +25,13 @@
   },
   "homepage": "https://github.com/weflex/react-baidu-map#readme",
   "devDependencies": {
-    "babel": "6.3.26",
-    "babel-cli": "6.3.17",
-    "babel-preset-es2015": "6.3.13",
-    "babel-preset-react": "6.3.13"
+    "babel-core": "^6.3.26",
+    "babel-loader": "^6.2.0",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "webpack": "^1.12.9"
+  },
+  "dependencies": {
+    "react": "^0.14.3"
```

Comment:
As far as what I read about `peerDependecies`, from [a NodeJS blogpost](https://nodejs.org/en/blog/npm/peer-dependencies/), it's mainly for solving complex dependencies issue rather than sharing dependencies between modules.